api/src/client/dynamics/license-status/RgbDynamicsBusinessAddressesClient.test.ts

```diff
@@ -9,8 +9,8 @@ jest.mock("axios");
 jest.mock("winston");
 const mockAxios = axios as jest.Mocked<typeof axios>;
 
-describe("DynamicsBusinessAddressClient", () => {
-  let client: BusinessAddressClient;
+describe("RgbDynamicsBusinessAddressClient", () => {
```

Comment:
dust - I'll only say it once, but I'm not a huge fan of acronyms in code names in general personally (Rgb) in this case, I generally would prefer long and descriptive names but I see that this is already pretty long so not a hill I want to die on

Reply:
Will update to RegulatedBusiness